Source/Scene/PostProcessStageTextureCache.js

```diff
@@ -245,7 +244,10 @@ function getFramebuffer(cache, stageName, dependencies) {
     pixelDatatype: pixelDatatype,
     clearColor: clearColor,
     stages: [stageName],
-    buffer: undefined,
+    buffer: new FramebufferManager({
+      pixelFormat: pixelFormat,
+      pixelDatatype: pixelDatatype,
```

Comment:
It doesn't looks like a `pixelFormat` and `pixelDatatype` are in the FramebufferManager constructor. What are your latest thoughts here - should they be part of the constructor or should they be passed through to `update`?

 CC https://github.com/CesiumGS/cesium/pull/9966#discussion_r770961733

Reply:
oops, I forgot to set them from constructor options. Even though this is maybe the only place that could benefit from both options, I think it doesn't hurt to have them since it simplifies the `update` arguments later. I'm for adding them as constructor options :+1: